libavformat/mxfdec.c

```diff
@@ -4220,6 +4423,9 @@ static const AVOption options[] = {
     { "eia608_extract", "extract eia 608 captions from s436m track",
       offsetof(MXFContext, eia608_extract), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1,
       AV_OPT_FLAG_DECODING_PARAM },
+    { "dovi_metadata_extract", "extract Dolby Vision metadata",
+      offsetof(MXFContext, dovi_metadata_extract), AV_OPT_TYPE_BOOL, {.i64 = 1}, 0, 1,
```

Comment:
We want to default this on for base FFmpeg?

Reply:
Initially my thinking was yes, but eia608_extract is disabled by default, and it's good to be consistent.